### PR TITLE
Removes virus symptom neutering

### PR DESCRIPTION
**Pull request by Time-Green**
## About The Pull Request
Removes virus neutering. This mechanic allows you to neuter symptoms to optimize stats without having to actually having those symptoms do anything

## Why It's Good For The Game
Virology has disease stats that influence how viruses behave. Do you want to buff your viruses resistance? Headaches gives +4 resistance, but makes your virus less desirable. So neuter it and take all the positives without having to compromise in the slightest!

I'm not saying we need to add deadly symptoms, but just... add nothing? I don't think the mix-and matching is also particularly engaging (I'll get on removing the thresholds...)

Virology is already barely designed, but this mechanic exists only to entirely circumvent the decision making process behind mixing and matching your diseases. It had the upside of giving the virologist more to do 7 years ago, but that's no longer a concern and we can instead work towards making virology interesting in a different way

## Changelog
:cl:
del: Removes virus symptom neutering
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
Virology is already an extremely effortless method to grief the crew via spamming unneutered negative symptoms. Plenty, and I mean plenty, of dipshits make the excuse to us that their one positive symptom in the virus is justification for the various negatives that come with it. Negatives that typically get worse the more powerful the virus is.

By making that the actual gameplay of virology, you've thrust the problem of it being a hollow system squarely into our laps as admins who now have to decide whether or not the dumbasses who fed their hydration healing virus a shitton of virus food to throw in every negative stat booster they could is playing virology earnestly or just a griefer. And that's not fun for anyone.

**Comment by PapaMichae1** [comment]
I'd prefer a full removal of viro over this

**Comment by the-orange-cow** [comment]
This is just going to make the crew miserable if anyone tries to interact with the viro system.  Very poorly thought out PR.

**Comment by Time-Green** [comment]
@necromanceranne Hmmm, good point. Ideally people don't put those viruses toghether with mass-spreading symptoms to keep them elective. People possibly it as an excuse to grief isn't very great and I don't wish to make admins work harder. It's a bit hard to quantify what "might" happen though, and neutering and thresholds were added at the same point so there's never been one without the other.

Still, I'd rather see people making less contagious diseases and maybe we can add some more stat symptoms with less lethal consequences? I'd prefer to see how it goes first

**Comment by zoomachina** [comment]
Currently, there's a choice for a station-sided virus maker: either put many good symptoms or put few while making the rest bad (neutered). The former option gives a number of positive effects, but low stats (no thresholds, other viruses will overpower yours), while the latter only allows having 1-2 effects, but stronger ones with better competition against other viruses. 

The simple fact is that no one in their right mind should put a negative symptom if they don't want the station to suffer. Because of that, the main effect of this change is killing the latter option.
There is no "decision making" involving putting active bad symptoms on a "good" virus. Not only would you struggle to find a threshold that's worth taking at the cost of a bad symptom, the virus maker shouldn't make that choice for the rest of the players, and anyone attempting to do so should be lynched.

In conclusion, this removes an intelligent choice that virus makers can make, leaving the right choice and the horrible choice instead, and this isn't good for the game.

**Comment by Time-Green** [comment]
> I'd prefer a full removal of viro over this

Me too buddy



> This is just going to make the crew miserable if anyone tries to interact with the viro system. Very poorly thought out PR.
> 
> Edit: also your 'why its good for the game' section feels lacking. Creating anti quality of life purely so someone will come along one day and add more interesting features is a tacit admission that you are making the game worse in a way that requires someone else to fix it later.

This is also the reason I kept this out of the virology removal PR, but we're now in a state where we don't need to have a full jobs content since it's no longer a job. Torturing yourself with disease making is completely elective (and it's realy not _that_ torturous, but the all in one mega healing viruses are gonna be significantly weaker. 

This will probably be the state of virology for a while, or even indefinitely. For me this is just breaking it down so I can built it up again, but I can get hit by a truck tomorrow and never make a PR again (players rejoice)

**Comment by necromanceranne** [comment]
> Still, I'd rather see people making less contagious diseases and maybe we can add some more stat symptoms with less lethal consequences? I'd prefer to see how it goes first

Viruses symptoms have a few categories. Currently, the vast majority of stat boosting ones do one of four things;
A) Nothing, its just a stat boost. Viral Acceleration is one of those.
B) Alter your character in some way. Alopecia is one of these.
C) Actively attack you.
D) Actively disrupt play if severe enough

Three of these four symptoms are grounds for escalation or alternatively become escalation problems. As a policy, we don't enforce optimization. With a change like this, we may well have too with regards to viruses based on how readily it will cause the creator to start drawing conflict towards themselves with even the most benign symptoms. A role shouldn't be regularly escalating with others if it is otherwise possible to play as a non-antagonist.

It would be okay if there were more non-affecting symptoms that boosted stats, but that would undermine this PR's purpose entirely. If the goal is to have upsides with downsides, just giving virologists more outs is self defeating.

One possible (and I'm still not sure it is quite enough) solution might be that neutering remains, but rather than remove the symptoms effects, instead simply prevents stat scaling and limit stage scaling. Many of the negatives don't actively do much harm when only at their lowest scale. Sneezing, headaches and coughing are just emote/chat spam at this stage. That way, the virus could be somewhat visible and a bit annoying, but not actively hitting administrative issues.

**Comment by Time-Green** [comment]
@necromanceranne and @zoomachina are correct. I'd prefer virology to mostly get dropped for the time being, but the reality is it is there and we have to deal with players touching it since we can't outright remove it.

I'll think of a neutering replacement. I'll close it tomorrow but in the meantime I'd love to hear suggestions for a way to make this better

**Comment by Timberpoes** [comment]
I brought up in the coding channels and cross-posting here for posterity.

If you're hitting thresholds on the positives, you're likely hitting them on the negatives too. A lot of negative symptoms have outright antagonistic thresholds and aren't designed around being simultaneously active on a beneficial virus without defeating the beneficial nature of such virus.

**Comment by Time-Green** [comment]
I have realised that the solution is to also remove thresholds muhahahhahaha

**Comment by Higgin** [comment]
Too much loss of depth and unintended consequences, not a fan without a larger rebalancing or change to how symptoms work. Neutering is med's good and bad tool inside the system to tweak on viruses - which have already been balanced around six symptoms with neutering possible and with the whole self-curing system acting on advanced viruses with any positive severity.

**Comment by Time-Green** [comment]
I like https://github.com/tgstation/tgstation/pull/83459 , wanna see how that plays first. If I come back with this change, I'll merge it with https://github.com/tgstation/tgstation/pull/83458 to also properly remove thresholds

**Comment by Time-Green** [comment]
Leaving open for a few more hours

**Comment by optimumtact** [comment]
you can close your own pr at any time

**Comment by Time-Green** [comment]
> you can close your own pr at any time

its called gatheringh feedback
